### PR TITLE
Improve Pivotal row in Production Setup table

### PR DESCRIPTION
**Pull request by astrieanna**
- Provider column should contain company name
- Change check marks to product links to match other rows

Co-authored-by: @dpb587-pivotal

### CODE REVIEW

File: content/en/docs/setup/_index.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ The following production environment solutions table lists the providers and the
...
 | [Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)](https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm)          | &#x2714;             |   &#x2714;     |             |      |          |
 | [oVirt](https://www.ovirt.org/)         |              |        |             |     | &#x2714;          |
-| [Pivotal Container Service](https://pivotal.io/platform/pivotal-container-service)         |              | &#x2714;       | &#x2714;           |      |          |
+| [Pivotal](https://pivotal.io/) | | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service) | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service)           |      |          |
```
Comment (sftim) [inline content/en/docs/setup/_index.md]: Should this page also have links to Cloud PKS and Essential PKS?

https://content.pivotal.io/blog/meet-the-pks-family-flexibility-for-your-kubernetes-architectures says:
> All three products are available from Pivotal and VMware.

Comment (astrieanna) [reply content/en/docs/setup/_index.md]: I didn't see a `pivotal.io` landing page for the other two products. On the page you linked, the Cloud PKS and Essential PKS links go to VMware.

Comment (sftim) [reply content/en/docs/setup/_index.md]: How about: list those things, but don't make them be hyperlinks?
(none of the options feels ideal, I'm trying to recommend a good compromise)

**Comment by sftim** [comment]
/lgtm
There might be a better way; anyway, this is an improvement.

**Pull request merged**